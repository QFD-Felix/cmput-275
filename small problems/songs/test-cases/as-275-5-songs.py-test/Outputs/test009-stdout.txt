4 0 0 8 6 2 10 1 12 6 0 1 1 8 10 4 10 0 0 0 7 0 1 0 0 0 0 0 0
