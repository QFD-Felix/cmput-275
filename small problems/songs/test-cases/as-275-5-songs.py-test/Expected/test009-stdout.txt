4 0 0 8 6 2 10 1 12 6 0 1 1 8 12 4 12 5 0 0 9 2 3 0 0 0 0 0 0 0 0
